packages/vats/src/core/boot-chain.js

```diff
@@ -0,0 +1,46 @@
+// @ts-check
+import { makeBootstrap } from './lib-boot.js';
+
+import * as basicBehaviorsPlus from './basic-behaviors.js';
+import { CHAIN_BOOTSTRAP_MANIFEST } from './chain-behaviors.js';
+import * as chainBehaviorsPlus from './chain-behaviors.js';
+import * as utils from './utils.js';
+
+export const MANIFEST = CHAIN_BOOTSTRAP_MANIFEST;
+
+const {
+  BASIC_BOOTSTRAP_PERMITS: _5,
+  PowerFlags: _3,
+  makeMyAddressNameAdminKit: _4,
+  ...basicBehaviors
+} = basicBehaviorsPlus;
```

Comment:
the numbers seem significant but aren't. consider:
```suggestion
const {
  BASIC_BOOTSTRAP_PERMITS: _B,
  PowerFlags: _P,
  makeMyAddressNameAdminKit: _m,
  ...basicBehaviors
} = basicBehaviorsPlus;
```